__tests__/app.test.js

```diff
@@ -166,4 +166,61 @@ describe('/api/articles/:article_id/comments', () => {
                 expect(comments.length).toBe(0);
             });
     });
+    test('POST:201 inserts comment into db and sends posted comment back to client', () => {
+        const newComment = {
+            username: 'rogersop',
+            body: 'very nice laptop'
+        };
+        return request(app)
+            .post('/api/articles/2/comments')
+            .send(newComment)
+            .expect(201)
+            .then((response) => {
+                const { comment } = response.body;
+                expect(comment.comment_id).toBe(19);
+                expect(comment.body).toBe('very nice laptop');
+                expect(comment.article_id).toBe(2);
+                expect(comment.author).toBe('rogersop');
+                expect(comment.votes).toBe(0);
+                expect(typeof comment.created_at).toBe('string');
+            });
+    });
+    test('POST:400 sends error message when given valid but non-existent article id', () => {
+        const newComment = {
+            username: 'rogersop',
+            body: 'very nice laptop'
+        };
+        return request(app)
+            .post('/api/articles/99/comments')
+            .send(newComment)
+            .expect(400)
+            .then((response) => {
+                expect(response.body.msg).toBe('bad request');
+            });
+    });
+    test('POST:400 sends error message when given invalid article id', () => {
```

Comment:
Nitpick, but this test description should be a valid but non existent article_id. This is because when we say invalid, we are usually referring to the data type of a variable not being correct